### PR TITLE
Cleanup ocr2keepers v2 package split

### PR DESCRIPTION
**Pull request by infiloop2**
Corresponding ocr2keepers PR https://github.com/smartcontractkit/ocr2keepers/pull/233

In evm20 only refer to ocr2keepers/v2
in evm21 only refer to ocr2keepers/v3
in delegate.go/utils.go (shared files) make imports explicit for version

ocr2keepers/config pkg was changed to be v2/config and v3/config. Use v2/config where config was being used for now and make the name explicit

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2keeper/evm21/encoder.go [hunk 1]
```diff
@@ -16,7 +15,6 @@ var (
 )
 
 type EVMAutomationEncoder21 struct {
-	encoding.BasicEncoder
```
Comment (infiloop2) [inline core/services/ocr2/plugins/ocr2keeper/evm21/encoder.go:19]: this wasn't used

File: core/scripts/chaincli/handler/keeper_deployer.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
 
-	offchain "github.com/smartcontractkit/ocr2keepers/pkg/config"
+	offchain20config "github.com/smartcontractkit/ocr2keepers/pkg/v2/config"
```
Comment (ferglor) [inline core/scripts/chaincli/handler/keeper_deployer.go:22]: 🏄 A shorter alias would be preferable, like maybe `v2config`/`configv2`?

File: core/services/ocr2/delegate.go [hunk 3]
```diff
@@ -17,11 +17,12 @@ import (
...
-	ocr2keepersconfig "github.com/smartcontractkit/ocr2keepers/pkg/config"
-	ocr2keepers20coordinator "github.com/smartcontractkit/ocr2keepers/pkg/coordinator"
-	ocr2keepers20polling "github.com/smartcontractkit/ocr2keepers/pkg/observer/polling"
-	ocr2keepers20runner "github.com/smartcontractkit/ocr2keepers/pkg/runner"
+	ocr2keepers20 "github.com/smartcontractkit/ocr2keepers/pkg/v2"
+	ocr2keepers20config "github.com/smartcontractkit/ocr2keepers/pkg/v2/config"
+	ocr2keepers20coordinator "github.com/smartcontractkit/ocr2keepers/pkg/v2/coordinator"
+	ocr2keepers20polling "github.com/smartcontractkit/ocr2keepers/pkg/v2/observer/polling"
+	ocr2keepers20runner "github.com/smartcontractkit/ocr2keepers/pkg/v2/runner"
+	ocr2keepers21config "github.com/smartcontractkit/ocr2keepers/pkg/v3/config"
```
Comment (ferglor) [inline core/services/ocr2/delegate.go:25]: 🏄 Same for all these? But totally fine if you'd prefer this way instead 👍

**Review by ferglor — APPROVED** [review]

File: core/scripts/chaincli/handler/keeper_deployer.go (hunk 2 above)
Comment (infiloop2) [reply core/scripts/chaincli/handler/keeper_deployer.go:22]: yep, this is just keepers file so will update

**Review by amirylm — APPROVED** [review]

File: core/services/ocr2/delegate.go (hunk 3 above)
Comment (infiloop2) [reply core/services/ocr2/delegate.go:25]: discussed offline, prefer to keep it verbose as this is shared with other products (ocr2vrf etc)

File: core/scripts/chaincli/handler/keeper_deployer.go (hunk 2 above)
Comment (infiloop2) [reply core/scripts/chaincli/handler/keeper_deployer.go:22]: discussed offline, will keep as is to have consistency with delegate.go

**Pull request merged**